packages/gui/src/constants/WalletConnectCommands.tsx

```diff
@@ -462,6 +526,7 @@ const walletConnectCommands: WalletConnectCommand[] = [
     command: 'transferNFT',
     label: <Trans>Transfer NFT</Trans>,
     service: ServiceName.WALLET,
+    bypassConfirm: true,
```

Comment:
I think including this as a command that doesn't always require confirmation was probably a mistake. I'm checking with E-ming.

Reply:
Updated